static-metric/Cargo.toml

```diff
@@ -19,7 +19,7 @@ lazy_static = "1.4"
 
 [dev-dependencies]
 criterion = "0.3"
-prometheus = { version = "0.11.0", path = "../" }
+prometheus = { version = "0.12.0", path = "../" }
```

Comment:
I guess I'll have to drop the `cargo package static-metric` test in CI otherwise we end up in a situation which is impossible to satisfy.